src/Utilities/copy.jl

```diff
@@ -24,17 +24,14 @@ function automatic_copy_to(
     copy_names::Bool = true,
     filter_constraints::Union{Nothing,Function} = nothing,
 )
-    if MOI.supports_incremental_interface(dest, copy_names)
-        return default_copy_to(dest, src, copy_names, filter_constraints)
-    elseif supports_allocate_load(dest, copy_names)
-        return allocate_load(dest, src, copy_names, filter_constraints)
-    else
+    if !MOI.supports_incremental_interface(dest, copy_names)
```

Comment:
One could argue whether we want to simplify this now and just having the fallback
```julia
copy_to(...) = default_copy_to(...)
```

Reply:
Yeah. Let's merge this first and then re-assess all the copy stuff.